mappings/net/minecraft/block/entity/CreakingHeartBlockEntity.mapping

```diff
@@ -0,0 +1,51 @@
+CLASS net/minecraft/unmapped/C_tkqzsaas net/minecraft/block/entity/CreakingHeartBlockEntity
+	FIELD f_bsnmvbck UPDATE_INTERVAL_VARIANCE I
+	FIELD f_bwukcwqb updateCooldown I
+	FIELD f_cvfzwxtz PLAYER_DETECTION_RANGE I
+	FIELD f_cwvstxzb GROWTH_SEARCH_MAX_ITERATIONS I
+	FIELD f_ehzrfpkl HURT_CALL_COUNT I
+	FIELD f_grilakjj CREAKING_ROAMING_RADIUS I
+	FIELD f_hlwtxpld MIN_UPDATE_INTERVAL I
+	FIELD f_ilcnsrbn lastConnectedCreakingPos Lnet/minecraft/unmapped/C_vgpupfxx;
+	FIELD f_kebdbxxl connectedCreaking Lcom/mojang/datafixers/util/Either;
+	FIELD f_kmnsfoas comparatorOutput I
+	FIELD f_mjtvqmci HURT_CALL_PARTICLE_TICKS I
+	FIELD f_mmycfyrt VERTICAL_SPAWN_RANGE I
+	FIELD f_mxplpjim MAX_SQUARED_DISTANCE_TO_CREAKING I
+	FIELD f_nsxjvugv HURT_CALL_INTERVAL I
+	FIELD f_okledrbs HORIZONTAL_SPAWN_RANGE I
+	FIELD f_pperjfcu NO_CONNECTED_CREAKING Ljava/util/Optional;
+	FIELD f_rmipaiio particleAndGrowthCooldown I
+	FIELD f_rnzvkqzx GROWTH_SEARCH_MAX_DEPTH I
+	FIELD f_tfhcwwsd ALLOWED_UNLOADED_CREAKING_TICKS I
+	FIELD f_tgjlbnaj SPAWN_ATTEMPTS I
+	FIELD f_xsvwxixc PARTICLE_AND_GROWTH_INTERVAL I
+	FIELD f_xyizqhss ticksLoaded J
+	METHOD m_aobtifjk tryGrowResin ()Ljava/util/Optional;
+	METHOD m_crfbayyq setConnectedCreakingUuid (Ljava/util/UUID;)V
+	METHOD m_eyesyodx calculateComparatorOutput ()I
+	METHOD m_jfayrjth unsetConnectedCreaking ()V
```

Comment:
```suggestion
	METHOD m_jfayrjth clearConnectedCreaking ()V
```
unless there's a good reason for the usage of unset here, i didn't reference the code